Dragonflight/WarriorProtection.lua

```diff
@@ -891,7 +890,7 @@ spec:RegisterAbilities( {
     heroic_leap = {
         id = 6544,
         cast = 0,
-        cooldown = function () return 45 + (talent.bounding_stride.enabled and -15 or 0) + (talent.wrenching_impact.enabled and 45 or 0) end,
+        cooldown = function () return 45 + (talent.bounding_stride.enabled and -15 or 0) end,
```

Comment:
Formatting for braces in addon code is to space them out from a consistency standpoint.
i.e.  `( talent.bounding_stride.enabled and -15 or 0 )`
In this case, I would do:  `return talent.bounding_stride.enabled and 30 or 45`

Reply:
Fixed in https://github.com/Wyste/hekili/commit/101bcd4e90bfb6ffe25dd8e3fb43c7209307c34e